os_stub/cryptlib_openssl/pk/x509.c

```diff
@@ -2401,6 +2401,8 @@ bool libspdm_set_attribute_for_req(X509_REQ *req, uint8_t *req_info, size_t req_
  * @param[in]      requester_info        requester info to gen CSR
  * @param[in]      requester_info_length The len of requester info
  *
+ * @param[in]       is_ca                if ture,set basic_constraints ture; Otherwise, set to false.
```

Comment:
here change to ' if **true**, set basic_constraints: **CA**: ture ', more accurate.